<launch>
 <!-- Serial Arduino Node -->
  <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="/dev/ttyACM0"/>
    <param name="baud" value="57600"/>
  </node>

 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" 
	name="jetson_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <param name="axis_linear" value="3" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value="1.0" type="double"/>
  <param name="scale_angular" value="1.0" type="double"/>

  <node pkg="jetson_joystick" 
	type="jetson_joystick_node" 
	name="jetson_teleop"/>

 <!-- Webcam Node -->
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="320" />
    <param name="image_height" value="240" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
    <param name="framerate" value="23"/>
  </node>
</launch>
